Mathlib/Data/FunLike/Embedding.lean

```diff
@@ -86,29 +87,29 @@ The second step is to add instances of your new `MyEmbeddingClass` for all types
 Typically, you can just declare a new class analogous to `MyEmbeddingClass`:
 
 ```
-structure CoolerEmbedding (A B : Type*) [CoolClass A] [CoolClass B]
-  extends MyEmbedding A B :=
-(map_cool' : toFun CoolClass.cool = CoolClass.cool)
+structure CoolerEmbedding (A B : Type*) [CoolClass A] [CoolClass B] extends MyEmbedding A B :=
+  (map_cool' : toFun CoolClass.cool = CoolClass.cool)
 
 section
 set_option old_structure_cmd true
```

Comment:
This option doesn't exist in Lean 4: can this be removed or should it be updated?